core/string_utils.cpp

```diff
@@ -131,6 +132,8 @@ UString jsonnet_string_unescape(const LocationRange &loc, const UString &s)
 
                     case 't': r += '\t'; break;
 
+                    case 'v': r += '\v'; break;
```

Comment:
I think this is fine because all it will do is convert "\v" to "\\v" when translating to contexts where the \ is meaningful and we need to escape it.  "\\v" in JSON is fine (it's not necessary but also it's not harmful).  I don't actually even think this is actually even used for JSON strings, it's used for Jsonnet strings (e.g. in the formatter) where you will need to do this.

Reply:
On the other hand if we *don't* add \v to Jsonnet then we shouldn't add this either.

Reply:
I have updated to remove parsing of `\v` (as you mentioned it will cause output to be invalid JSON)